core/src/test/scala/unit/kafka/server/ListOffsetsRequestTest.scala

```diff
@@ -180,7 +183,8 @@ class ListOffsetsRequestTest extends BaseRequestTest {
 
     TestUtils.generateAndProduceMessages(servers, topic, 10)
 
-    assertEquals((-1L, -1), fetchOffsetAndEpoch(firstLeaderId, 0L, 0))
+    assertEquals((0L, -1), fetchOffsetAndEpoch(firstLeaderId, ListOffsetRequest.EARLIEST_TIMESTAMP, 0))
+    assertEquals((10L, -1), fetchOffsetAndEpoch(firstLeaderId, ListOffsetRequest.LATEST_TIMESTAMP, 0))
```

Comment:
Would it make sense to test `EARLIEST_TIMESTAMP`, `LATEST_TIMESTAMP`, and `0L` cases for all the versions?